src/app/utils/otp.ts

```diff
@@ -17,6 +17,18 @@ export const generateOtp = (): string => {
     .join('')
 }
 
+/**
+ * Randomly generates and returns a 3 letter OTP prefix.
+ * @returns 3 letter OTP prefix string
+ */
+export const generateOtpPrefix = (): string => {
+  // Generates cryptographically strong pseudo-random data. 65 is the starting ASCII character code for upper case letters.
```

Comment:
Nice code! I smiled at using cryptographically strong randomness here, which seemed unnecessary 🙂. The cost is low though, so let's keep it!